Gems/ROS2/Code/Source/RobotImporter/SDFormat/Hooks/ROS2AckermannModelHook.cpp

```diff
@@ -0,0 +1,320 @@
+                if (transformInterface)
+                {
+                    return AZ::Success(transformInterface->GetWorldTM().GetTranslation());
+                }
+            }
+
+            return AZ::Failure(AZStd::string("Failed to obtain position parameter for wheel entity."));
+        }
+
+        float GetTrack(const AZ::Entity* entityLeft, const AZ::Entity* entityRight)
+        {
+            const auto positionLeft = GetPosition(entityLeft);
+            const auto positionRight = GetPosition(entityRight);
+
+            if (positionLeft.IsSuccess() && positionLeft.IsSuccess())
+            {
+                return positionLeft.GetValue().GetDistance(positionRight.GetValue());
+            }
+
+            return 0.0f;
+        }
+
+        bool IsSteeringWheel(const std::string& jointNameSteering, const std::string& jointNameWheel, const sdf::Model& sdfModel)
+        {
+            const auto jointPtrSteering = sdfModel.JointByName(jointNameSteering);
+            const auto jointPtrWheel = sdfModel.JointByName(jointNameWheel);
+            if (jointPtrSteering != nullptr && jointPtrWheel != nullptr)
+            {
+                return (jointPtrSteering->ChildName() == jointPtrWheel->ParentName());
+            }
+
+            return false;
+        }
+
+        float GetWheelRadius(AZ::Entity* entity)
+        {
+            if (entity != nullptr)
+            {
+                PhysX::EditorColliderComponent* colliderComponent = entity->FindComponent<PhysX::EditorColliderComponent>();
+                if (colliderComponent != nullptr)
+                {
+                    float radius = 0.0f;
+                    entity->Activate();
+                    if (entity->GetState() == AZ::Entity::State::Active)
+                    {
+                        PhysX::EditorPrimitiveColliderComponentRequestBus::EventResult(
+                            radius,
+                            AZ::EntityComponentIdPair(entity->GetId(), colliderComponent->GetId()),
+                            &PhysX::EditorPrimitiveColliderComponentRequests::GetCylinderRadius);
+                        entity->Deactivate();
+                    }
+                    return radius;
+                }
+            }
+
+            return 0.0f;
+        }
+
+        float GetWheelRadius(AZ::Entity* entityLeft, AZ::Entity* entityRight)
+        {
+            const float radiusLeft = GetWheelRadius(entityLeft);
+            const float radiusRight = GetWheelRadius(entityRight);
+
+            if (fabsf(radiusLeft - radiusRight) < epsilon)
+            {
+                return radiusLeft;
+            }
+            else
+            {
+                AZ_Warning(
+                    "ROS2AckermannModelPluginHook",
+                    false,
+                    "VehicleConfiguration parsing error: left and right wheel radii does not match. (%f and %f)",
```

Comment:
Cap the float precision to the epsilon width for better readability

```suggestion
                    "VehicleConfiguration parsing error: left and right wheel radii does not match. (%.4f and %.4f)",
```